### DISCUSSION TITLE
hello world!

### DISCUSSION THREAD

**Discussion by clickercookie** [clickercookie/clickercookie.github.io#45] [Announcements]
hey it's me @FifthTundraG comin' to you live from the clickercookie account because i can't enable discussions on my main account lol.

in case it wasn't obvious this is just a test announcement lol. i'll make a real one in a little while

**Comment by FifthTundraG** [comment]
ok i looked everywhere and it turns out unless the repo is a part of an organization only the owner can make announcements :sob:
